keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/partials/client-detail.html

```diff
@@ -282,6 +282,14 @@
                 <kc-tooltip>{{:: 'root-url.tooltip' | translate}}</kc-tooltip>
             </div>
 
+            <div class="form-group clearfix block" data-ng-show="protocol == 'wsfed'">
```

Comment:
This part was not present in your previous PR. Is it related to another PR you would like to submit regarding cloudtrust/keycloak-wsfed#44?

Reply:
Yeah, sorry about that. Too many changes waiting :-)
Should be better now